outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
```

Comment:
I feel like packages vs. modules doesn't align with the actual content of this section, which is more like "when to use packages"